### PR TITLE
some fixes i made that fix mostly all of the crash and issues 

### PR DESCRIPTION
**Pull request by CypherNebula84**
i added
print("                  	Contributor: Alex (CypherNebula84) - CypherNebula84@outlook.com")
 If you want, I can remove it, but I fixed some general things like it crashing when invalid characters are entered.
 I added it after, but there are a couple more improvements, like now it says:
 "Select a machine (number), create a new one with 'c', or enter 0 to quit:"
 So you can create stuff now. It asks if you want to configure, and it refreshes so it can be detected.
 But not too much changed. I was going to make a logo, but I'm not so good at it.
sorry for last time i should of made a stable guide on what i changed instead of just fix and working
i tested it it doesn't crash and it dose what its supposed to do idk mac compatibility but just figured i would give you a improved version if its even improved but yha hope you like it or whatever have a good day

### CODE REVIEW

**Comment by CypherNebula84** [comment]
and i just fixed one error i forgot to test

**Comment by CypherNebula84** [comment]
and im now fixing my mistakes now

**Comment by CypherNebula84** [comment]
im done

**Comment by CypherNebula84** [comment]
are you adding?

**Comment by DDXofficial** [comment]
Please be patient, I am dealing with other things outside this project at the moment. I will merge when I have the time.

File: launcher.py [hunk 1]
```diff
@@ -13,14 +18,15 @@ def clear_screen():
 clear_screen()
 
 # VERSION
-launcher_version = str("0.2-pre")
+launcher_version = str("0.1a")
```
Comment (DDXofficial) [inline launcher.py:21]: Please do not change the version number.

**Review by DDXofficial — CHANGES_REQUESTED** [review]

File: launcher.py (hunk 1 above)
Comment (CypherNebula84) [reply launcher.py:21]: i don't remember changing

File: launcher.py [hunk 2]
```diff
@@ -84,53 +84,47 @@ def path_text():
...
+
+                if configure_new_machine == 'y':
+                    # Configure the new machine
+                    print("Starting 86Box for configuration...")
+                    os.system(app_path + f" -S \"{machine_path}/{new_machine_name}/86box.cfg\"")
+
+                # Reload the script to detect the new machine
+                print("Reloading the script...")
+                script_path = os.path.abspath(__file__)
+                os.execv(sys.executable, [sys.executable, script_path])
```
Comment (DDXofficial) [inline launcher.py:115]: What OS did you test this on? I'm getting path-related issues on Windows.

Comment (CypherNebula84) [reply launcher.py:115]: oh i tried on win 11

Comment (CypherNebula84) [reply launcher.py:115]: i dont have mac rn

Comment (CypherNebula84) [reply launcher.py:115]: idk

Comment (CypherNebula84) [reply launcher.py:115]: i figured id make a better ver i guess its better to update with the 4.1

Comment (CypherNebula84) [reply launcher.py:115]: i guess config doesn't work in 4.1

Comment (CypherNebula84) [reply launcher.py:115]: its basically useless rn without the config and the only crash i have is if i enter invalid name in create but

Comment (CypherNebula84) [reply launcher.py:115]: oh i guess it does work it justs takes 5 min

Comment (CypherNebula84) [reply launcher.py:115]: https://github.com/DDXofficial/sl86/assets/154773078/6fc7e4cd-05c2-4e87-b179-b9e951dbee96

**Comment by CypherNebula84** [comment]
not best quality i had compression because max is 10mb and it was higher and zip file is suspicious

**Pull request merged**